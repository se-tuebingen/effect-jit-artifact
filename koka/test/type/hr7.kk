// ,("(undefined :: some a. [a -> a] -> Int) (undefined :: some c. [(forall d. d -> c) -> c])", Wrong)

fun ok() {
  (throw("hi") : some<a> list<a -> (forall<b> b -> b)> -> exn int)(throw("there") : some<c> list<(forall<d> d -> d) -> c>)
}