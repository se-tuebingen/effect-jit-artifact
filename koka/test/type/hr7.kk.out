type/hr7/ok : () -> exn int